Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewControllerTests.swift

```diff
@@ -0,0 +1,685 @@
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+import Prelude
+import SnapshotTesting
+import UIKit
+
+final class NoShippingPledgeViewControllerTests: TestCase {
+  private let userWithCards = GraphUser.template |> \.storedCards .~ UserCreditCards(
+    storedCards: [
+      UserCreditCards.visa,
+      UserCreditCards.masterCard
+    ]
+  )
+
+  override func setUp() {
+    super.setUp()
+    AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    UIView.setAnimationsEnabled(false)
+  }
+
+  override func tearDown() {
+    AppEnvironment.popEnvironment()
+    UIView.setAnimationsEnabled(true)
+
+    super.tearDown()
+  }
+
+  func testView_PledgeContext_UnavailableStoredCards() {
+    let response = UserEnvelope<GraphUser>(me: self.userWithCards)
+    let mockService = MockService(fetchGraphUserResult: .success(response))
+    let project = Project.template
+      |> \.availableCardTypes .~ [CreditCardType.discover.rawValue]
+
+    orthogonalCombos(Language.allLanguages, Device.allCases).forEach { language, device in
+      withEnvironment(apiService: mockService, currentUser: User.template, language: language) {
```

Comment:
Will you turn the no shipping flag on in the environment for these tests? I'm not seeing the headings in the snapshots and I'm guessing that's why. (Also, there are a lot of them so I'm totally fine with doing minimal tests in this file if you'd rather and flesh it out once we get rid of the other tests).